src/js/electricPianoBand.js

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.chartAuthoring.electricPianoBand", {
+        gradeNames: ["flock.band"],
+
+        components: {
+            midiNoteSynth: {
+                type: "floe.chartAuthoring.midiNoteSynth"
+            },
+
+            pianoEnvelopeSynth: {
+                type: "floe.chartAuthoring.pianoEnvelopeSynth"
+            },
+
+            carrierSynth: {
+                type: "floe.chartAuthoring.carrierSynth"
+            }
+        }
+    });
+
+    fluid.defaults("flock.interconnectOutputSynth", {
+        gradeNames: ["flock.modelSynth"],
+
+        outputBus: "@expand:{flock.enviro}.busManager.acquireNextBus(interconnect)",
+
+        synthDef: {
+            ugen: "flock.ugen.out",
+            bus: "{that}.options.outputBus",
+            expand: 1,
+            sources: "{that}.options.ugenDef"
+        }
+    });
+
+    fluid.defaults("floe.chartAuthoring.midiNoteSynth", {
+        gradeNames: ["flock.interconnectOutputSynth"],
+
+        ugenDef: {
+            id: "freq",
+            ugen: "flock.ugen.midiFreq",
+            rate: "control",
+            note: {
+                id: "noteSequencer",
+                ugen: "flock.ugen.sequencer"
+            }
+        },
+        addToEnvironment: "head"
+    });
+
+    fluid.defaults("floe.chartAuthoring.carrierSynth", {
+        gradeNames: ["flock.modelSynth"],
+
+        ugenDef: {
+            ugen: "flock.ugen.sinOsc",
+            phase: 0.0,
+            freq: {
+                ugen: "flock.ugen.in",
+                bus: "{midiNoteSynth}.options.outputBus"
+            },
+
+            mul: {
+                ugen: "flock.ugen.in",
+                bus: "{pianoEnvelopeSynth}.options.outputBus"
+            }
+        },
+
+        phases: [0.0, 0.5, 0.25],
+
+        synthDef: {
+            ugen: "flock.ugen.sum",
+            sources: {
+                expander: {
+                    funcName: "floe.chartAuthoring.carrierSynth.additiveExpander",
+                    args: ["{that}.options.phases", "{that}.options.ugenDef"]
+                }
+            }
+        }
+    });
+
+    floe.chartAuthoring.carrierSynth.additiveExpander = function (phases, ugenDef) {
```

Comment:
The audible effect is that you have three sine tones running out of phase with each other, creating a more complex "reedy" tone. It's a very, very rudimentary model of an electric piano like a Fender Rhodes.

Yes, this expander's output is just an array of unit generators that someone will have to know what to do with.

I was thinking that perhaps we should turn this into an expander that:

1. Can take an array or hash of arrays representing values that should be used to produce unit generators for additive synthesis
2. Automatically wrap these in the necessary <code>flock.ugen.sum</code> unit generator definition

Does that seem sensible and address your concern?